include/lsst/daf/base/DateTime.h

```diff
@@ -63,15 +63,15 @@ class DateTime {
     explicit DateTime(double date, DateSystem system = MJD, Timescale scale = TAI);
     DateTime(int year, int month, int day, int hr, int min, int sec,
              Timescale scale = TAI);
-    explicit DateTime(std::string const& iso8601);
+    explicit DateTime(std::string const& iso8601, Timescale scale);
 
     long long nsecs(Timescale scale = TAI) const;
     double get(DateSystem system = MJD, Timescale scale = TAI) const;
-    std::string toString(void) const;
+    std::string toString(Timescale scale) const;
```

Comment:
Correct. Most of DateTime assumes TAI as the default, and there was a lot of existing code that assumed `toString` was returning a TAI date if the DateTime had been defined in TAI. So this was an invasive change, but it exposed broken code. @ktlim specifically requested no default and I agree with him.